src/test/java/com/javaproject/restservice/repository/UserRepositoryTest.java

```diff
@@ -0,0 +1,37 @@
+package com.javaproject.restservice.repository;
+
+import static com.javaproject.restservice.entity.User.*;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
+
+import com.javaproject.restservice.entity.*;
```

Comment:
try and be more specific when importing something where you only us one of the classes from the libarary